app/src/components/data-grid/GenericGridColumnDefinitions.tsx

```diff
@@ -297,7 +302,11 @@ export const GenericCommentColDef = <T extends GridValidRowModel>(props: {
       return (
         <Tooltip title={params.value} arrow>
           <span>
-            <IconButton aria-label="observation-subcount-comment" disabled>
+            <IconButton
+              aria-label="observation-subcount-comment"
```

Comment:
nit: label is specific when it should be generic

Reply:
Fixed